andela/balder-ah-frontend
# #163600063 Display users articles on AH homepage

#### What does this PR do?
Fetches and displays fresh articles from remote API on AH homepage.

#### Description of Task to be completed?
Have user articles show up on homepage

#### How should this be manually tested?
```bash
git clone git@github.com:andela/balder-ah-frontend.git

cd balder-ah-frontend

npm install

npm run start:dev
```

Visit `/` to see user articles displayed on the homepage.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories? (If applicable)
[#163600063](https://www.pivotaltracker.com/story/show/163600063)

#### Screenshots (If applicable)
<img width="1047" alt="screen shot 2019-01-31 at 1 42 16 pm" src="https://user-images.githubusercontent.com/5154605/52054847-13bfc800-255e-11e9-9542-e258b57291de.png">

#### Questions:
N/A
